libs/authorizenet/driver/magento/src/authorize-net.service.ts

```diff
@@ -22,7 +22,10 @@ export class DaffMagentoAuthorizeNetService implements DaffAuthorizeNetService {
 		return new Observable(observer =>
 			Accept.dispatchData(transformMagentoAuthorizeNetRequest(paymentTokenRequest, this.config), (response: AuthorizeNetResponse) => {
 				if (response.messages.resultCode === 'Error') {
-					throw new Error(response.messages.message[0].code + ':' + response.messages.message[0].text);
+          const MappedError = DAFF_AUTHORIZE_NET_ERROR_CODE_MAP[response.messages.message[0].code];
```

Comment:
Needs a test covering at least one case.

Reply:
The driver service doesn't have a unit test, I'll add one

Reply:
I added a test but messing with the `Accept` global doesn't seem to work in `beforeEach` so I added it each `it`.